manifests/dispatch_router.pp

```diff
@@ -0,0 +1,52 @@
+# == Class: capsule::dispatch_router
+#
+# Install and configure Qpid Dispatch Router
+#
+# === Parameters:
+#
+#
+class capsule::dispatch_router (
+) {
+
+  class { 'qpid::router': }
+
+  qpid::router::ssl_profile { 'router-ssl':
+    ca   => $certs::ca_cert,
+    cert => $certs::params::messaging_client_cert,
+    key  => $certs::params::messaging_client_cert,
```

Comment:
I guess I need some advice here.  For the qpid dispatcher, I need 2 ssl profiles on the hub - one for the server and one for the connection to the existing broker.

I'm not really sure what to use, I don't want to create yet more certificates, can I use the qpid-broker ones?